blog/content/post/2023-12-14-aem_phase3.md

```diff
@@ -0,0 +1,318 @@
+---
+title: TrenchBoot Anti Evil Maid - Phase 3
+abstract: "This blog post marks completion of next phase of TrenchBoot Anti Evil
+           Maid project for Qubes OS. Even though user experience didn't change
+           too much, the implementation went through a major overhaul."
+cover: /covers/trenchboot-logo.png
+author: krystian.hebel
+layout: post
+published: true
+date: 2023-12-14
+archives: "2023"
+
+tags:
+  - QubesOS
+  - TrenchBoot
+  - GRUB2
+  - Xen
+  - Hypervisor
+categories:
+  - Firmware
+  - Bootloader
+  - Hypervisor
+  - OS Dev
+  - Security
+
+---
+
+## Introduction
+
+If you haven't read the previous [TrenchBoot AEM blog
+post](https://blog.3mdeb.com/2023/2023-01-31-trenchboot-aem-for-qubesos/) or
+our [project plan](https://docs.dasharo.com/projects/trenchboot-aem-v2/) yet,
+it may be a good lecture to catch up on what the TrenchBoot AEM project is
```

Comment:
Was the use of the word `lecture` intended here?  Such use seems unexpected to me.

Reply:
I copied this from previous post, it even doesn't link to it, but I agree that this isn't proper use of this word. It probably comes from Polish `lektura`, which means `reading` (as a noun). Changed in https://github.com/3mdeb/news-and-ideas/pull/561/commits/d87984353d98b14ec17191cbad5b122f2ec048fa